library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileSplitter.java

```diff
@@ -53,6 +53,7 @@
    * This is a threshold on the no. of blocks emitted per window. A lot of blocks emitted
    * per window can overwhelm the downstream operators. This setting helps to control that.
    */
+  @NotNull
```

Comment:
As per discussions on pull request #283 we decided to make blocksThreshold as compulsory parameter, so user always sets it.